pymongo/database.py

```diff
@@ -292,6 +292,147 @@ def get_collection(
             read_concern,
         )
 
+    def _get_encrypted_fields(self, kwargs, coll_name, ask_db):
+        encrypted_fields = kwargs.get("encryptedFields")
+        if encrypted_fields:
+            return encrypted_fields
+        if (
+            self.client.options.auto_encryption_opts
+            and self.client.options.auto_encryption_opts._encrypted_fields_map
+            and self.client.options.auto_encryption_opts._encrypted_fields_map.get(
+                f"{self.name}.{coll_name}"
+            )
+        ):
+            return self.client.options.auto_encryption_opts._encrypted_fields_map[
+                f"{self.name}.{coll_name}"
+            ]
+        if ask_db:
+            colls = list(self.list_collections(filter={"name": coll_name}, **kwargs))
+            if colls and colls[0]["options"].get("encryptedFields"):
+                return colls[0]["options"]["encryptedFields"]
+        return None
+
+    @_csot.apply
+    def create_encrypted_collection(
```

Comment:
This should be defined on ClientEncryption, not database: https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.rst#id13

Reply:
Done.